app/src/main/java/com/iitp/njack/iitp_connect/core/home/AuthViewModel.java

```diff
@@ -14,18 +16,24 @@
  */
 public class AuthViewModel extends ViewModel {
 
-    private static final FirebaseAuthLiveData firebaseAuthLiveData = new FirebaseAuthLiveData();
+    private AuthRepository loginRepository;
+    private FirebaseAuthLiveData firebaseAuthLiveData;
```

Comment:
Get this using Dagger also if possible?

Reply:
The repository is getting FirebaseAuthLiveData using dagger.The viewModel is getting repository using dagger.So viewModel is automatically getting FirebaseAthLiveData through dagger.